packages/presets/src/editors/editor-container.ts

```diff
@@ -154,10 +163,10 @@ export class AffineEditorContainer
     return html`${keyed(
       this.model.id,
       this.mode === 'page'
-        ? html`<doc-editor
+        ? html`<affine-doc-editor
```

Comment:
I think `doc-editor` and `affine-doc-editor` need to be distinguished in a better way. What do you think? @doodlewind

Reply:
Maybe we can just load fragments in `editor-container` and no need to create `affine-doc-editor`?

Reply:
I agree... "Hardest thing in programming is naming variables" :P

I just used the naming in the github issue:-
https://github.com/toeverything/blocksuite/issues/5757#issue-2045838265
![image](https://github.com/toeverything/blocksuite/assets/54364088/511124ca-8c87-43f9-bd2b-1697d6c519ca)